webapp/packages/plugin-administration/src/ConfigurationWizard/ServerConfiguration/IServerConfigurationFormPartState.ts

```diff
@@ -24,6 +24,8 @@ const ServerConfigurationFormPartStateConfigSchema = schema.object({
   serverName: schema.string().optional(),
   serverURL: schema.string().optional(),
   sessionExpireTime: schema.number().optional(),
+  secureCookies: schema.boolean().optional(),
+  supportedHosts: schema.array(schema.string()),
```

Comment:
Let's use `supportedHosts` as `string` so we will join hosts when getting data in the form part (load), and we will split it back to the list when saving, so we don't need to produce a complex editable computed pattern in UI